python_modules/dagster-graphql/dagster_graphql/schema/errors.py

```diff
@@ -68,15 +68,45 @@ class Meta:
     def __init__(self, error_info):
         super().__init__()
         self._error_info = check.inst_param(error_info, "error_info", SerializableErrorInfo)
-        self.message = error_info.message
-        self.stack = error_info.stack
-        self.cause = error_info.cause
-        self.context = error_info.context
-        self.className = error_info.cls_name
+        self._message = error_info.message
+        self._stack = error_info.stack
+        self._className = error_info.cls_name
+        self._cause = error_info.cause
+        self._context = error_info.context
+
+    def resolve_message(self, _graphene_info):
+        check.invariant(
+            isinstance(self, GraphenePythonError),
+            f"GraphenePythonError methods called on a {type(self)} - this usually indicates that "
+            "a SerializableErrorInfo was passed in where a GraphenePythonError was expected",
+        )
```

Comment:
you would think this could be a helper method, but when this happens you can't use other methods!